clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStatsReporter.java

```diff
@@ -97,24 +98,6 @@ protected void registerStats() {
             0,
             VERSION_TOPIC_END_OFFSET_REWIND_COUNT));
 
-    // System store mostly operates on hybrid partial updates so batch metrics are not useful.
-    if (!VeniceSystemStoreUtils.isUserSystemStore(storeName)) {
```

Comment:
So it is intentional to add these metrics back for sys stores, right?

Reply:
No, the new location of this code still has the same condition check; it's in line 140/123.